database/manager.go

```diff
@@ -0,0 +1,181 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package database
+
+import (
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"go.uber.org/zap"
+)
+
+//
+// DatabaseManager writes all committed keys to the database on a timer.
+//
+
```

Comment:
This line is necessary to prevent the go auto-formatter from cropping the empty comment lines. I wanted to make it very obvious what parts of this file corresponded to what types.